crates/oxc_cli/src/options.rs

```diff
@@ -0,0 +1,30 @@
+use std::path::PathBuf;
+
+use clap::ArgMatches;
+
+pub struct CliOptions {
+    pub quiet: bool,
+    pub paths: Vec<PathBuf>,
+}
+
+impl<'a> TryFrom<&'a ArgMatches> for CliOptions {
+    type Error = &'a str;
+
+    fn try_from(matches: &ArgMatches) -> Result<Self, Self::Error> {
+        let mut paths = vec![];
+
+        for path in matches.get_many::<PathBuf>("path").unwrap() {
+            let globbed = glob::glob(&path.to_string_lossy())
+                .expect("Failed to read glob pattern")
```

Comment:
nitpick: this should not panic, return Err instead?

Reply:
Yeah, I inherited this code from the glob PR, I can refactor it